### PR TITLE
Prediction for example kaggle_ndsb1

### PR DESCRIPTION
**Pull request by Gelu74**
This is related to https://github.com/dmlc/mxnet/issues/1143 and @antinucleon example on using mxnet on Kaggle Data Science Bowl 1 image classification.

I have added prediction and the generation of submission.csv so the model output can be submitted to kaggle. I have also done minor changes to training to be in the same format as image-classification examples (maybe the whole folder could be placed inside image-classification just as another example).

### CODE REVIEW

File: example/kaggle-ndsb1/bn2.conf [hunk 1]
```diff
@@ -0,0 +1,216 @@
+
```
Comment (antinucleon) [inline example/kaggle-ndsb1/bn2.conf]: probably we don't need this cxxnet conf file?

File: example/kaggle-ndsb1/bn.conf [hunk 2]
```diff
@@ -0,0 +1,216 @@
+
```
Comment (antinucleon) [inline example/kaggle-ndsb1/bn.conf]: probably we don't need this cxxnet conf file?

**Comment by Gelu74** [comment]
yes, sorry, I made a commit to my fork yesterday as I plan to port those files to mxnet, (I am new to github and I thought pull request would use my repository in the state when the pull request was done)
This is the repository state that should be pulled:
https://github.com/Gelu74/mxnet/tree/d9fd588d676f0be66b050b0ab76ac06f1a7359d5/example/kaggle-ndsb1
Do you want me to revert the last commit 72f8e2164f0b881110d573fd68cd179090b2fc13 in my fork?

**Comment by antinucleon** [comment]
Thanks for your work. A simpler way is switch to a new branch, then modify commit, eg, remove files, then squash commits (https://mxnet.readthedocs.org/en/latest/contribute.html#how-to-combine-multiple-commits-into-one). After that we can send a PR. After merge it, you are safe to delete the branch, and your master branch won't be affected. Maybe we can close this PR and open a new PR from a new branch?

**Comment by Gelu74** [comment]
OK, close this one and I'll make a new PR in a few hours

**Pull request closed without merging**